docs/openapi/pipeline.yaml

```diff
@@ -9889,8 +9916,10 @@ components:
                             description: "the secret name of the azure access information"
                             type: string
                             example: "myaccesssecretname"
-                azure:
+                aks:
```

Comment:
Is this really for AKS only and not for any distribution running on Azure?

Reply:
It should be `azure`, but the API currently gives back `aks` and this PR just fixes that in the openapi descriptor. We should address the `aks` -> `azure` transition in another PR.